ROADMAP.md

```diff
@@ -215,14 +215,18 @@ Inclusive of the projects, public-good services, and standardized interfaces, Si
 
 ## Client SDKs
 
-### Short Term (< 6 mo)
+### Completed in 2024
+
+* Releases
+  * [sigstore-java v1 release](https://blog.sigstore.dev/announcing-sigstore-java-1-0/) of signing and verifying API, along with Maven and Gradle plugins
+  * [sigstore-go beta release](https://blog.sigstore.dev/sigstore-go-signing-beta/) of signing and verifying API
+  * [sigstore-python v3 release](https://github.com/sigstore/sigstore-python/releases/tag/v3.0.0) supporting attestations and trusted roots
+  * [cosign v2.4.0 release](https://github.com/sigstore/cosign/releases/tag/v2.4.0) supporting protocol buffer bundles and conformance testing
+
+### Remaining in 2024
 
 * Releases:
-    * sigstore-java v1 release
-    * sigstore-go beta release
     * sigstore-rust v1 release
-    * sigstore-python v2 release
-* FFI Design
```

Comment:
Is this because we've seen a lack of interest from the registries in non-native implementations?

Reply:
It's definitely true that many registries strongly prefer native implementations, but also I don't think anyone has been working in this area? I know https://github.com/sigstore/sigstore-rs is picking up steam, but I think we would need someone to focus specifically on FFI support?

Reply:
Yea, Zack N was looking into this previously. Ultimately, until we have a GA release of sigstore-rs, I don't think FFI is an option.